src/incubator/WheelPicker/index.tsx

```diff
@@ -13,6 +13,12 @@ import Text, {TextProps} from '../../components/text';
 
 const AnimatedFlatList = Animated.createAnimatedComponent(FlatList);
 
+export enum WheelPickerAlign {
```

Comment:
Consider setting this enum as a static member of the component. 
So people can easily do `WheelPicker.aligments.LEFT` for instance